TestownikConsoleApp/QuestionWriter.cs

```diff
@@ -1,35 +1,25 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 
 namespace TestownikConsoleApp
 {
     public class QuestionWriter
     {
+        readonly Random rnd = new Random();
         public void WriteQuestions()
         {
             QuestionParser questionParser = new QuestionParser();
             var questions = questionParser.ReadQuestions("Pytania");
-
-            //foreach (QuestionDto question in questions)
-            //{
-            //    Console.WriteLine(question.Question);
-            //    foreach (AnswerDto answer in question.Answers)
-            //    {
-            //        Console.WriteLine(answer.Answer);
-            //    }
-            //}
-
-            questions.ForEach(question =>
+            var countOfQuestions = questions.Count();
+            var indexOfRndQuestion = rnd.Next(countOfQuestions);
```

Comment:
co się stanie jak będziesz miał np 20 elementów w liście, count of question bedzie wtedy 20 i wylosuje ci 20? A następnie bedziesz wyciagać element o indeksie 20?

Reply:
Będzie nadal działać ;)